src/port1.0/portconfigure.tcl

```diff
@@ -138,7 +138,7 @@ default compiler.fortran_fallback    {[portconfigure::get_fortran_fallback]}
 # define options
 commands configure autoreconf automake autoconf xmkmf
 # defaults
-default configure.env       ""
+default configure.env       "DEVELOPER_DIR = /Library/Developer/CommandLineTools"
```

Comment:
What's the compatibility of this with older OS versions? Do we set this on Linux as well? @kencu

Not that I expect from @satraul to have access to older versions, or test on every single version, but it would be helpful if someone could confirm that things don't break on older boxes.